mavros/src/lib/enum_to_string.cpp

```diff
@@ -158,7 +158,6 @@ static const std::array<const std::string, 29> mav_type_strings{{
 /* 25 */ "VTOL reserved 5",               // VTOL reserved 5
 /* 26 */ "Onboard gimbal",                // Onboard gimbal
 /* 27 */ "Onboard ADSB peripheral",       // Onboard ADSB peripheral
-/* 28 */ "Dodecarotor",                   // Dodecarotor
 }};
```

Comment:
`pip install -U pymavlink`

Reply:
I have the latest pymvalink and this was thr resulting enun